usr/bin/ublue-nix-install

```diff
@@ -95,6 +95,29 @@ echo "experimental-features = nix-command flakes" | sudo tee -a /etc/nix/nix.con
 
 sleep 1
 
+# nb stands for nix backup
+
+echo "Installing nix backup"
+
+sudo mkdir /opt/nb
```

Comment:
I'd rather this had a more meaningful name like `/opt/nixbackup`

Reply:
yeah you could change it to that but I did nb since it is very short